lib/page/assets/lock/LockDetailPage.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:polka_wallet/common/components/checkRule.dart';
+import 'package:polka_wallet/common/components/formulaInput.dart';
+import 'package:polka_wallet/common/components/goPageBtn.dart';
+import 'package:polka_wallet/common/components/subTitle.dart';
+import 'package:polka_wallet/page/assets/lock/LockResultPage.dart';
+import 'package:polka_wallet/store/app.dart';
+import 'package:polka_wallet/utils/i18n/index.dart';
+
+class LockDetailPage extends StatefulWidget {
+  LockDetailPage(this.store);
+
+  static final String route = '/assets/lock/detail';
+  final AppStore store;
+
+  @override
+  _DetailPageState createState() => _DetailPageState(store);
+}
+
+class _DetailPageState extends State<LockDetailPage> {
+  _DetailPageState(this.store);
+
+  final AppStore store;
+
+  @override
+  Widget build(BuildContext context) {
+    var dic = I18n.of(context).assets;
+    
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(dic['lock.tokens'])
+      ),
+      body: SafeArea(
+        child: Builder(
+          builder: (BuildContext context) {
+          return ListView(
+            shrinkWrap: true, 
+            padding: const EdgeInsets.all(20),
+            children: <Widget>[
+              Text(
+                dic['transaction.message'],
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 10),
+                child: Text(
+                  dic['lock.transaction.instruction'],
+                  textAlign: TextAlign.center,
+                ),
+              ),
+              subTitle(dic['formula']),
+              Row(
+                children: <Widget>[
+                  formulaInput(
+                    lable: dic['lock.duration']
+                  ),
+                  Text(' , '),
+                  Expanded(
```

Comment:
On this 1st "Lock Tokens" and also on the 1st "Signal Tokens" pages, the selection box for "Signal,Lock" does not need to exist here, since the value of "Lock" should be hard-coded in the "Your Transaction Message" section of the Lock Tokens page, and its value of "Signal" should be hard-coded in the "Your Transaction Message" section of the Signal Tokens page.